src/Native/build-native.sh

```diff
@@ -90,6 +90,16 @@ build_native()
         echo "Failed to build corefx native components."
         exit 1
     fi
+    
+    if [ "$__BuildOS" != "OSX" ]; then
+        echo "Verifying System.Security.Cryptography.Native.OpenSsl.so dependencies"
```

Comment:
Hmm, it would actually be very ugly using the custom command unless I created a separate bash script to do the verification. It doesn't seem to be worth the hassle.